package.json

```diff
@@ -102,11 +102,11 @@
   "dependencies": {
     "@bsull/augurs": "^0.6.0",
     "@emotion/css": "^11.10.6",
-    "@grafana/data": "^11.3.0",
+    "@grafana/data": "^11.6.0",
     "@grafana/lezer-logql": "^0.2.7",
-    "@grafana/runtime": "^11.3.0",
+    "@grafana/runtime": "^11.6.0",
```

Comment:
Do we need to update the grafanaDependency version in the plugin.json now?

Reply:
Oh I thought you wanted grafana/ui, data, runtime updated as well. Should I revert all those updates?

Reply:
No, I think this is correct, but if we are using assets from Grafana 11.6 that could cause issues if users are running Grafana < 11.6?

Reply:
Yeah, the panel menus are show on hover, and more importantly the fields combobox detected_fields call is broken on 11.3. The fields call might work in 11.4, but the panel menus will be broken until 11.5 or 11.6.